Suggestion to include accounts created with typos in this page, and to mention that the customers should export / import DNS records (#9410)
cloudflare/cloudflare-docs: content/fundamentals/get-started/basic-tasks/manage-domains/move-domain.md

* Lose access to your email address or Cloudflare account.

> * Registered a Cloudflare account with a typo in your emai.

Reviewer: ```suggestion
* Registered a Cloudflare account with a typo in your email.
```

Before transferring an active Cloudflare domain to another Cloudflare account, you must remove any [DNSSEC configurations](/dns/dnssec/) and [add-ons or subscriptions](/fundamentals/account-and-billing/account-maintenance/cancel-subscription/).

> We also recommend you to export the DNS records of your zone(s) before adding each zone in the new account, so that you may import the correct DNS records in the new account when adding each zone in the new account. You may learn how to export/import your DNS records [here](https://developers.cloudflare.com/dns/manage-dns-records/how-to/import-and-export/).

Reviewer: ```suggestion
We also recommend [exporting](/dns/manage-dns-records/how-to/import-and-export/#export-records) the DNS records of your zone while it is in the previous account. Then, you can [import](/dns/manage-dns-records/how-to/import-and-export/#import-records) the correct DNS records into the new account.
```

> If you miss this step Cludflare will automaticly import your proxied dns records and you should find an 1000 error. If that happends, please follow this [tutorial](https://community.cloudflare.com/t/fixing-error-1000-dns-points-to-prohibited-ip/178709) to fix this issue. 

Reviewer: ```suggestion
If you miss this step, Cloudflare will import your proxied dns records might experience a [1000 error](/support/troubleshooting/cloudflare-errors/troubleshooting-cloudflare-1xxx-errors/). 
```
